### ISSUE TITLE
Web: Host dashboard main - Property Management System (PMS)

### ISSUE THREAD

**Issue by Timmhart** [domits1/Domits#149]
## 🚀 Feature Summary

A property management system (PMS) for short-term rentals centralizes daily operations across listings, guests, and teams. It manages reservations, pricing, housekeeping, payments, and guest communication from one platform. The system streamlines automation, improves operational efficiency, and ensures a consistent guest experience at scale.

----

Main issues: [PMS](https://github.com/domits1/Domits/issues/149), [CRS](https://github.com/domits1/Domits/issues/501), [CM](https://github.com/domits1/Domits/issues/440), [RMS](https://github.com/domits1/Domits/issues/436), [Calendar & Prices](https://github.com/domits1/Domits/issues/36), [Inbox](https://github.com/domits1/Domits/issues/767), [Finance](https://github.com/domits1/Domits/issues/163), [Integrations](https://github.com/domits1/Domits/issues/433)
Use Case / Scenario Testing: [PMS](https://github.com/domits1/Domits/issues/2262), [CRS](https://github.com/domits1/Domits/issues/2265), [CM](https://github.com/domits1/Domits/issues/2266), [RMS](https://github.com/domits1/Domits/issues/2267), [Calendar & Prices](https://github.com/domits1/Domits/issues/2299), [Inbox](https://github.com/domits1/Domits/issues/2291), [Finance](https://github.com/domits1/Domits/issues/2287)

----

<img width="1306" height="517" alt="Image" src="https://github.com/user-attachments/assets/33ca2634-7ae0-421b-813d-6478c0be11fa" />

----

Host and property onboarding:
- [ ] #182

----

Website: Host dashboard main - Property Management System (PMS) - Property API
Tabs/functions:
- [ ] #150
- [ ] #154 
- [ ] #501
- [ ] #151 
- [ ] #510
- [ ] #503
- [ ] #1084
- [ ] #506
- [ ] #153  
- [ ] #589
- 
- [ ] Later: #502
- [ ] Later: #505
- [ ] Later: #152 
- [ ] Later: #1477
- [ ] Later: #504
- [ ] Later: #507
- [ ] Later: #508
-
- [ ] Later: #155 
- [ ] Later: #509
-
- [ ] #1694
-

---

## 🧩 Problem Statement
What problem does this feature solve? Who is affected, and why does it matter? > Focus on the problem, not the solution.

No streamlined property management operations

---

## 🎯 Desired Outcome

Streamlined property management operations

---

## 👥 Target Users
Who benefits from this feature?
- [ ] Guests / Customers
- [x] Hosts / Property Managers / Enterprise
- [ ] Admin / Internal users
- [ ] Developers
- [ ] Other: _______

---

## 📈 Business & User Impact
How valuable is this feature?

- **Impact level:**
  - [ ] Low (nice-to-have)
  - [ ] Medium (improves workflows)
  - [ ] High (unblocks key use cases)
  - [x] Critical (strategic / revenue-impacting)

- **User reach:**
  - [ ] Few users
  - [ ] Many users
  - [x] All users
  - [ ] Unknown

---

## ⚠️ Constraints & Considerations
- Performance
- Security
- Privacy
- Accessibility
- Backward compatibility
- External dependencies

---

## 🧪 Acceptance Criteria / User Stories
How can we validate that this feature is complete?

**User Story**
As a host,
I want a comprehensive dashboard to manage my property listings, bookings, and communications,
so that I can efficiently run my rental business and respond to guests' needs.

**Acceptance Criteria**
- [ ] Overview Summary (Dashboard tab):
The dashboard displays a summary of key metrics, including total bookings, upcoming reservations, and revenue earned for the current month.
Visual graphs show booking trends over time, such as occupancy rates and revenue changes.

- [ ] Property Listings Management (Listings tab):
Hosts can view all their active listings with quick access to key information (e.g., occupancy status, price, and guest rating).
An “Edit” button for each listing allows for quick adjustments to pricing, availability, and property details.

- [ ] Booking Management (Reservations tab):
A dedicated section shows all current and upcoming bookings, including guest names, check-in/check-out dates, and payment status.
Hosts can click on each booking to view details, including guest messages, special requests, and cancellation options.

- [ ] Guest Communication (Messaging tab):
An integrated messaging system allows hosts to communicate directly with guests before and during their stay.
Notifications alert hosts of new messages, upcoming check-ins, and special requests from guests.

- [ ] Financial Overview (Finance tab):
Hosts can access a financial summary detailing earnings from bookings, including breakdowns of fees, taxes, and payout schedules.
A downloadable report option allows hosts to generate financial statements for accounting or tax purposes.

- [ ] Calendar and Availability Management (Calendar tab):
A calendar view shows all bookings, blocked dates, and availability at a glance, allowing easy updates to property availability.
Hosts can quickly block off dates for personal use or maintenance.

- [ ] Performance Analytics:
Hosts receive insights into property performance, including booking rates, guest ratings, and feedback.
Comparison tools allow hosts to benchmark their listings against similar properties in the area.

- [ ] Support Resources:
A “Help” section provides access to resources, FAQs, and contact information for customer support.
A link to community forums or tips for successful hosting is also available.

- [ ] Action Items and Reminders:
A section for action items alerts hosts about tasks needing attention (e.g., responding to guest inquiries, updating listings).
Calendar reminders for upcoming bookings or tasks related to property maintenance or cleaning.

**What-If Scenarios**

- [ ] What if a host receives a last-minute booking?
Provide instant notifications and an option to quickly prepare for the new guest, with reminders for check-in procedures.

- [ ] What if a host needs to cancel a booking?
Allow hosts to cancel bookings with clear guidelines on potential penalties and automated messages to inform the guest of the cancellation.

- [ ] What if a guest leaves a negative review?
Enable hosts to respond to reviews directly on the dashboard, providing them an opportunity to address concerns publicly and professionally.

- [ ] What if the host has multiple properties?
Include a filtering system or tabs to easily switch between properties and manage listings individually or collectively.

- [ ] What if the host wants to adjust prices for peak seasons?
Allow bulk editing of pricing for specific time frames, or set seasonal pricing automatically based on demand data.

- [ ] What if the host notices a sudden drop in bookings?
Provide alerts or insights on market trends, suggest potential price adjustments, or highlight any new competitors in the area.

- [ ] What if a host wants to analyze guest demographics?
Include analytics showing guest statistics (e.g., average age, geographical location) to help tailor marketing and enhance guest experiences.

- [ ] What if a host is unsure how to enhance their listing?
Offer personalized tips based on performance metrics and guest feedback to help hosts optimize their listings for better visibility and bookings.

- [ ] What if the host has questions about financial reports?
Include a dedicated financial support feature that provides assistance with understanding reports and tax implications, including a contact option for a financial advisor.

- [ ] What if a host has technical issues with the dashboard?
Provide a live chat or support ticket option within the dashboard for immediate assistance, along with a troubleshooting guide for common issues.
This Host Dashboard ensures that hosts have all the necessary tools and insights to manage their short-term rental business effectively, streamline operations, and enhance guest interactions.

-
Later
- [ ] Space management
- [ ] Multi property management
- [ ] POS

Resources:
https://www.altexsoft.com/blog/hotel-property-management-systems-products-and-features/

**Comment by iLLy-27** [comment]
- Payments => Reporting
- Revenue tab
- Dashboard => personal information + recent listing
- Review knop naar messaging

**Comment by stefanhopman1** [comment]
@iLLy-27 
Graag voor de host dashboard tabjes die er nog niet zijn “Coming Soon” in de grid…

- Reservations Coming Soon
- Revenues Coming Soon
- Occupancy/ADR Coming Soon
- Property Care Coming Soon
- IoT Hub Coming Soon
- Pricing Coming Soon
- Distribution Coming Soon
- Monitoring Coming Soon
- Screening Coming Soon
- Setup Coming Soon
- Promo Codes Coming Soon
Net zoals hier is geschreven: https://www.domits.com/guestdashboard/booking
Dan zijn deze pagina’s er alvast.
